cirq/sim/clifford/clifford_simulator.py

```diff
@@ -61,7 +61,7 @@ def __init__(self, seed: 'cirq.RANDOM_STATE_OR_SEED_LIKE' = None):
     @staticmethod
     def is_supported_operation(op: 'cirq.Operation') -> bool:
         """Checks whether given operation can be simulated by this simulator."""
-        if protocols.is_measurement(op): return True
+        if isinstance(op.gate, cirq.MeasurementGate): return True
```

Comment:
probably needs a TODO and issue to allow Pauli measurements in CliffordSimulator.

Reply:
Done.